keps/sig-node/4818-allow-zero-value-for-sleep-action-of-prestop-hook/README.md

```diff
@@ -0,0 +1,580 @@
+<!--
```

Comment:
since this is just a small additon to the previos KEP, I think it will be OK to add the OWNERS file: https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/3960-pod-lifecycle-sleep-action/OWNERS

Reply:
I've added the file. Can I know more about why this file is needed at a KEP level? I haven't seen it much before.